eng/Versions.props

```diff
@@ -4,9 +4,9 @@
     <ProductVersion>9.0.0</ProductVersion>
```

Comment:
Don't we need to change this to 9.0.20?

Reply:
Yeah we need, but removing it so it s autogenerated